code/src/main/java/com/onlinecontacttracing/messaging/EmailSender.java

```diff
@@ -0,0 +1,114 @@
+package com.onlinecontacttracing.messaging;
+
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.gmail.Gmail;
+import com.google.api.services.gmail.GmailScopes;
+import com.google.api.services.gmail.model.Label;
+import com.google.api.services.gmail.model.ListLabelsResponse;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+import javax.mail.internet.MimeMessage;
+import javax.mail.MessagingException;
+import java.util.logging.Logger;
+import com.onlinecontacttracing.storage.PersonEmail;
+import com.onlinecontacttracing.storage.NotificationBatch;
+import org.apache.commons.codec.binary.Base64;
+import com.google.api.services.gmail.model.Message;
+import java.io.IOException;
+import java.io.InputStream;
+import java.security.GeneralSecurityException;
+import java.io.ByteArrayOutputStream;
+import java.util.Properties;
+import javax.mail.Session;
+import javax.mail.internet.InternetAddress;
+import static com.googlecode.objectify.ObjectifyService.ofy;
+
+/**
+ * The email sender will load a list of emails from the notification batch
+ * and compile messages to send to each person. They will receive an email
+ * from cccoders@onlinecontacttracing.com.
+ */
+public class EmailSender {
+
+  private static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
+  private static final List<String> SCOPES = Collections.singletonList(GmailScopes.GMAIL_SEND);
+  private static final String SERVICE_ACCOUNT_ID = "online-contact-tracing@appspot.gserviceaccount.com";
+  private static final String EMAIL_TO_SEND_WITH = "cccoders@onlinecontacttracing.com";
+  private static final String CREDENTIALS_FILE_PATH = "online-contact-tracing-f798898872f4.p12";
+  private static final String TOKENS_DIRECTORY_PATH = "tokens";
+  private static final String APPLICATION_NAME = "Online Contact Tracing";
+  static final Logger log = Logger.getLogger(EmailSender.class.getName());
+
+  /**
+  * Sends emails with a email subject and a message to a list of contacts.
+  */
+  public static void sendEmailsOut(String emailSubject, CompiledMessage compiledMessage, String messageLanguage) {
```

Comment:
Please pass in the EmailSubject Object rather than passing in the string it returns.